### DISCUSSION TITLE
steam invalid ssl certificate

### DISCUSSION THREAD

**Discussion by Kencho5** [mitmproxy/mitmproxy#6486] [General]
Hello,

i've installed mitmproxy and the certificate on my pc. after entering steam through browser and steam desktop app everything works as expected. when i tried the same things from my friend's pc (following the same instructions) i get this error: 
![image](https://github.com/mitmproxy/mitmproxy/assets/53686164/d9bb2497-84e7-454d-8bc6-b0df904d6dee)

it works from browser without problems but i get this from the desktop app.

how could i fix this?

**Comment by TheK0tYaRa** [comment]
So far adding `--set ignore_hosts='(steampowered|steamcommunity|steamstatic|steamusercontent).com'` worked for me, writing that here for the future of others.

Although it has introduced a noticeable delay when opening the store tab.
